doc/en/index.html

```diff
@@ -0,0 +1,481 @@
+          </dt>
+          <dd>Quick tutorial introducing new users to common tasks.</dd>
+          <dt>
+           <a href="http://docs.geoserver.org/latest/en/user/tutorials/index.html">Tutorials</a>
+           ( <a href="http://docs.geoserver.org/stable/en/user/tutorials/index.html">stable</a>
+           | <a href="http://docs.geoserver.org/maintain/en/user/tutorials/index.html">maintenance</a> )
+          </dt>
+          <dd>Step by step tutorials exploring GeoServer functionality.</dd>
+          <dt>
+           <a href="http://docs.geoserver.org/latest/en/user/extensions/index.html">Extensions</a>
+           ( <a href="http://docs.geoserver.org/stable/en/user/extensions/index.html">stable</a>
+           | <a href="http://docs.geoserver.org/maintain/en/user/extensions/index.html">maintenance</a> )
+          </dt>
+          <dd>Optional modules providing additional functionality to the base GeoServer application.</dd>
+        </dl>
+
+        <h3>Developer Manual</h3>
+
+        <dl>
+          <dt>  
+          <a href="http://docs.geoserver.org/latest/en/developer/">Developer Manual</a>
+          ( <a href="http://docs.geoserver.org/stable/en/developer/">stable</a>
+          | <a href="http://docs.geoserver.org/maintain/en/developer/">maintenance</a> )
+          </dt>
+          <dd>
+            Project checkout and build instructions.
+          </dd>
+          <dt>
+           <a href="http://docs.geoserver.org/latest/en/developer/translation.html">Translating GeoServer</a>
+          </dt>
+          <dd>We would like to make GeoServer available in as many languages as possible. If you are able to help start here!</dd>
+          <dt>
+           <a href="http://docs.geoserver.org/latest/en/developer/policies/index.html">Polices and Procedures</a>
+          </dt>
+          <dd>Here is how the project operates - volunteers welcome!</dd>
+        </dl>
+
+        <h3>Documentation Manual</h3>
+        
+        <di>
+          <dt><a href="http://docs.geoserver.org/latest/en/docguide/">Documentation Guide</a></dt>
+          <dd>For those wishing to add/edit GeoServer documentation.</dd>
+          <dt><a href="http://docs.geoserver.org/latest/en/docguide/quickfix.html">Quickfix</a><dt>
+          <dd>Spot a spelling mistake or other quickfix? Here is how to make the change directly in GitHub.</dd>
+        </dl>        
+
+        <h3>Release Versions</h3>
+        <p>These are snapshots of the documentation at the time the version was released.
+           You can also download the version-specific documentation on the appropriate
+           <a href="http://geoserver.org/download">Download</a> page.</p>
+
+        <table>
+
+          <tr>
+            <th>Release</th><th>User Manual</th><th>Developer Manual</th>
+          </tr>
+          <tr>
+            <td><strong><a href="http://geoserver.org/release/2.13.0/">2.13.2</a></strong></td>
```

Comment:
this and some of the other urls below seem incorrect. eg. http://geoserver.org/release/2.13.2/ exists, so does http://geoserver.org/release/2.13.1/
```suggestion
            <td><strong><a href="http://geoserver.org/release/2.13.2/">2.13.2</a></strong></td>
```

Reply:
I left these as-is, sorry!

More importantly is probably missing all the 2.14-2.20 versioned URLs? 😬

Reply:
Not quite. All the 2.13.1, etc. links were generated manually. Since around 2.14, we've only been (automatically) generating `2.14.x` doc artifacts, so there's rather fewer new links required (6, instead of 20 or so). There's been a whole bunch of discussion in the PMC meetings and on the mailing lists on which of these archetypes to follow (docs for every point release vs. only major releases) and we've settled on the latter.
This'll also make updating it a good bit easier.